sinonjs/sinon
# Fix build and test bugs in #1498 and #1479

#### Purpose 
Should fix the errors we are seeing in #1498, introduced by a proxyquire config bug, as well as a case of a missing file causing trouble for the builds in #1479.

#### Background (Problem in detail)  - optional
See the discussion in #1498 and #1479.

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. Set the sauce lab credentials (or run [selenium-standalone](https://www.npmjs.com/package/selenium-standalone) locally to test with local browsers)
4. `npm test`
5. `npm run test-cloud`

Those steps cover #1498. 

To test that #1479 failed before, but not after, do this for on `master` and on this branch:
1. `rm -r pkg`
2. `npm test`
